public/manifest.json

```diff
@@ -0,0 +1,16 @@
+{
+    "short_name": "pillbug",
+    "name": "pillbug",
+    "icons": [
+        {
+            "src": "favicon-128.png",
+            "sizes": "128x128",
+            "type": "image/png"
+        }
+    ],
+    "start_url": "./",
+    "scope": ".",
+    "display": "standalone",
+    "theme_color": "#000000",
+    "background_color": "#000000"
+}
```

Comment:
I wonder if PWAs ignore any extra properties here; if so, manifest.json might be a good place for instances hosting pillbug to both customize their PWA branding as well as pillbug behavior if we read this file too